src/helpers/statusBarHelper.ts

```diff
@@ -34,5 +34,5 @@ export const isUsingButtonNavigation = () => {
   const deviceHeight = Dimensions.get('screen').height;
   const windowHeight = Dimensions.get('window').height;
   const bottomNavBarHeight = deviceHeight - windowHeight;
-  return bottomNavBarHeight > 70;
+  return bottomNavBarHeight > 95;
```

Comment:
gonna make a separate PR for this.. seems this is a good number to distinguish between gesture mode and button nav.. unless you have better ideas @christianbaroni

Reply:
https://github.com/rainbow-me/rainbow/pull/5460